cmd/app/add_test.go

```diff
@@ -144,32 +144,32 @@ func TestAppAddCommand(t *testing.T) {
 				appSelectMock.On("TeamAppSelectPrompt").Return(prompts.SelectedApp{Auth: mockAuthTeam1}, nil)
 
 				// Mock valid session for team1
-				cm.ApiInterface.On("ValidateSession", mock.Anything, mock.Anything).Return(api.AuthSession{
```

Comment:
note: acronyms should be capitalized - this is where most of the changes occurred because we consistently stuck with `Api`, `Http`, etc.